src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -68,6 +68,8 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
   // Update dynamic parameters when responses change
   useEffect(() => {
     Object.entries(parameterExpressions).forEach(([field, expression]) => {
+      if (field === 'validationInternal')
+        return setEvaluatedParameters((prevState) => ({ ...prevState, [field]: expression }))
```

Comment:
Don't need this -- it's already being taken care of in master using "internalParameters"

Reply:
oops

Reply:
merged by accident

Reply:
95ded6c07ae2811626a79ac97518536ac2efdfd1